_monitoring-plugins/ad/index.md

```diff
@@ -17,24 +17,21 @@ Anomaly detection  automatically detects anomalies in your OpenSearch data in ne
 
 You can pair the anomaly detection plugin with the [alerting plugin]({{site.url}}{{site.baseurl}}/monitoring-plugins/alerting/) to notify you as soon as an anomaly is detected.
 
-To use the anomaly detection plugin, your computer needs to have more than one CPU core.
-{: .note }
-
 ## Get started with Anomaly Detection
 
 To get started, choose **Anomaly Detection** in OpenSearch Dashboards.
-To first test with sample streaming data, choose **Sample Detectors** and try out one of the preconfigured detectors.
+To first test with sample streaming data, you can try out one of the preconfigured detectors with one of the sample datasets.
 
-### Step 1: Create a detector
+### Step 1: Define a detector
```

Comment:
The outline of AD page doesn't show the step titles.
![Screen Shot 2021-10-05 at 4 51 11 PM](https://user-images.githubusercontent.com/49084640/136120947-326d2e60-47b0-4576-b560-38296948b187.png)